iscript/src/iscript/mac.py

```diff
@@ -569,30 +570,34 @@ def filter_apps(all_paths, fmt, inverted=False):
 
 
 # create_all_notarization_zipfiles {{{1
-async def create_all_notarization_zipfiles(all_paths, path_attr="app_path"):
+async def create_all_notarization_zipfiles(all_paths, path_attr):
```

Comment:
Maybe name this `path_attrs` if it's a list?

Reply:
Yep, will do.